src/te_checks.c

```diff
@@ -253,16 +299,12 @@ struct check_result *check_perm_macro_class_mismatch(__attribute__((unused)) con
 			continue;
 		}
 
-		// ignore permissions 'something[_something]_$CLASSNAMEALIAS_perms'
-		if (class_alias && ends_with_suffix_perms(perms->string, perm_len, class_alias, class_alias_len)) {
-			continue;
-		}
-
 report:
 		return make_check_result('S', S_ID_PERM_SUFFIX,
-					 "Permission macro %s does not match class %s",
+					 "Permission macro %s does not match class %s%s",
```

Comment:
Wouldn't it look cleaner to put the space in the string here, rather than in the argument?

Reply:
changed the message format